### PR TITLE
Add resizable feature for field

### PR DESCRIPTION
**Pull request by Smiranin**
1) Add resizable feature for field. You can set it for Input `resizable` default = false;
2) Change placeholder http to https for links.

### CODE REVIEW

File: src/app/ng2wig/ng2wig.module.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { Ng2WigToolbarService } from './ng2wig-toolbar.service';
 
 @NgModule({
   declarations: [
-    Ng2WigComponent
+    Ng2WigComponent,
```
Comment (stevermeister) [inline src/app/ng2wig/ng2wig.module.ts]: ?

**Comment by stevermeister** [comment]
Hi Man! 
how are you doing? 

thank you for your PR 
but please hold on with new features, we are now working on big update (https://github.com/stevermeister/ngx-wig/tree/brand-new)

and discussion is here 
https://github.com/stevermeister/ngx-wig/issues/76

You are welcome to join! 
We are still looking for the best way to implement plugins (to have custom buttons)

**Comment by Smiranin** [comment]
Hi, my teacher!) 
I am fine and still working with angular and love it.

Ok, I will close this PR and make new PR in new branch later.

File: src/app/ng2wig/ng2wig.module.ts (hunk 1 above)
Comment (Smiranin) [reply src/app/ng2wig/ng2wig.module.ts]: )

**Comment by stevermeister** [comment]
@Smiranin you are welcome to update your PR

**Pull request closed without merging**